lib/src/messaging_client.dart

```diff
@@ -102,6 +102,7 @@ content={message: {senderId: 6187, serviceId: 88697, text: with one picture}, fi
         sentAt: envelope.publishedAt.toDateTime().millisecondsSinceEpoch,
         text: content['text'] ?? '',
         userId: content['senderId'],
+        messageId: content['messageId'] ?? int.parse(envelope.uuid.value),
```

Comment:
Hmm. Converting the UUID to an int here worries me, as that seems like something that could fail.

What do you think about changing `messageId` from an `int` to a `int?`? Since we only need the value for AI chat messages at the moment, maybe that's safe?